src/data/emacs-mode/agda2-mode.el

```diff
@@ -32,10 +32,10 @@ Note that the same version of the Agda executable must be used.")
       (require 'filladapt)
     (error nil))
   (unless (fboundp 'overlays-in) (load "overlay")) ; for Xemacs
-  (unless (fboundp 'propertize)                    ; for Xemacs 21.4
```

Comment:
I think this was here just to support older versions of Emacs, and so doesn't violate the 'namespace' conventions.  From a MELPA standpoint, this PR could probably be closed.  Will discuss more in https://github.com/melpa/melpa/pull/7080